modules/API/API.js

```diff
@@ -241,6 +246,18 @@ function initCommands() {
             logger.debug('Pin participant command received');
 
             const state = APP.store.getState();
+
+            // if id not provided, unpin everybody.
+            if (!id) {
+                const pinnedParticipants = getPinnedActiveParticipants(state);
```

Comment:
It looks to me that this will handle only the stage filmstrip use case. Am I wrong?

Reply:
Good catch!